src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -7,8 +7,9 @@
 
 namespace System
 {
-    public partial class AppDomain
+    public partial class AppDomain : System.MarshalByRefObject
     {
+        private const int DefaultADID = 1;
```

Comment:
Or maybe get rid of the const field altogether and just return `1` from `Id` to be consistent with the other properties that are returning constants without having const fields?